src/editors/hooks.js

```diff
@@ -8,7 +8,7 @@ import * as module from './hooks';
 
 export const initializeApp = ({ dispatch, data }) => useEffect(
   () => dispatch(thunkActions.app.initialize(data)),
-  [],
+  [data],
```

Comment:
I updated editor initialization to have dependency on data which would be `blockId, blockType, learningContextId,
 lmsEndpointUrl, studioEndpointUrl` so useEffect will run when `blockId` updates.

Outside of testing, I don't see these fields changing... so I think this change to the editor code would be fine.

Reply:
Yeah, this dependacy array is such that it is all values which are static, so I don;t think it would be too complex a behavior change under the hood. Might be worthwhile to smoke test in on devstack for sure.